conf/application.conf

```diff
@@ -91,19 +91,33 @@ controllers {
         needsAuditing = false
     }
 
-    uk.gov.hmrc.individualsmatchingapi.controllers.SandboxPrivilegedCitizenMatchingController = {
+    uk.gov.hmrc.individualsmatchingapi.controllers.MicroserviceHelloWorld = {
         needsLogging = false
         needsAuditing = false
     }
 
-    uk.gov.hmrc.individualsmatchingapi.controllers.SandboxPrivilegedIndividualsController = {
+    #### P1 Live Controllers
+
+    uk.gov.hmrc.individualsmatchingapi.controllers.LivePrivilegedCitizenMatchingController = {
```

Comment:
you can remove LivePrivilegedCitizenMatchingController and LivePrivilegedIndividualsController as they are true by default (I think)